app/src/main/java/org/fossasia/susi/ai/forgotpassword/ForgotPasswordActivity.kt

```diff
@@ -0,0 +1,148 @@
+package org.fossasia.susi.ai.forgotpassword
+
+import android.app.ProgressDialog
+import android.content.DialogInterface
+import android.content.Intent
+import android.graphics.Color
+import android.os.Bundle
+import android.support.v7.app.AppCompatActivity
+import android.view.MenuItem
+import android.view.View
+import kotlinx.android.synthetic.main.activity_forgot_password.*
+import org.fossasia.susi.ai.R
+import org.fossasia.susi.ai.helper.AlertboxHelper
+import org.fossasia.susi.ai.helper.Constant
+import org.fossasia.susi.ai.login.LoginActivity
+
+
+
+/**
+ * Created by meeera on 6/7/17.
+ */
+class ForgotPasswordActivity : AppCompatActivity (), ForgotPasswordView {
+
+    var forgetPasswordPresenter: IForgotPasswordPresenter?= null
+    var progressDialog: ProgressDialog? = null
+
+    override fun invalidUrl() {
```

Comment:
move methods starting from `invalidUrl` to `wrongEmail` after `onOptionsItemSelected`